packages/scvelo/meta.yaml

```
@@ -0,0 +1,17 @@
+name: scvelo
```

Is this just an ID? Otherwise, I'd change it to _scVelo_ as in the description.

---

I think so, let's find out.

```suggestion
name: scVelo
```